core/src/main/java/org/apache/accumulo/core/client/summary/CountingSummarizer.java

```diff
@@ -179,7 +179,7 @@ private void init(SummarizerConfiguration conf) {
      * @param consumer
      *          emit counter objects derived from key and value to this consumer
      */
-    void convert(Key k, Value v, Consumer<K> consumer);
```

Comment:
This looks like it shouldn't be removed.

Reply:
This was the questionable one.  It is not used though, which makes me question it.  Perhaps @keith-turner can weigh in since I believe he worked on this.

Reply:
This should not be removed.   The intent of this is to allow user to count arbitrary things about key/values.  The currently included impls only happen to count things about keys.

Reply:
OK so this is just a case where a test impl should be written for full test coverage.

Reply:
Opened #849 which could be a nice one for a new contributor.